.eslintrc.js

```diff
@@ -0,0 +1,41 @@
+module.exports = {
+    env: {
+        browser: true,
+        node: true,
+    },
+    extends: [
+        'eslint:recommended',
+        'plugin:@typescript-eslint/eslint-recommended',
+        'plugin:@typescript-eslint/recommended',
+        'prettier',
+        'prettier/@typescript-eslint',
+        'plugin:prettier/recommended',
+    ],
+    parser: '@typescript-eslint/parser',
+    parserOptions: {
+        ecmaVersion: 11,
+        sourceType: 'module',
+    },
+    plugins: ['@typescript-eslint'],
+    ignorePatterns: ['dist/', 'reports/'],
+    rules: {
+        'prettier/prettier': ['error', { endOfLine: 'auto' }],
+    },
+    overrides: [
+        {
+            files: ['*.js'],
+            rules: {
+                '@typescript-eslint/no-var-requires': 'off',
+            },
+        },
+        {
+            files: ['*.ts'],
+            rules: {
```

Comment:
I think these should just go in rules, not in overrides

Reply:
Ok moved.